crates/pacdef/src/main.rs

```diff
@@ -10,8 +10,7 @@
     clippy::unwrap_used,
     clippy::use_debug,
     clippy::use_self,
-    clippy::wildcard_dependencies,
-    missing_docs
```

Comment:
Keep the lints. If you'd rather not write the docstrings yourself I will take care of them.